zipkin-lens/src/components/App/App.jsx

```diff
@@ -39,29 +40,31 @@ const App = () => {
       <UiConfig>
         <MuiPickersUtilsProvider utils={MomentUtils}>
           <ThemeProvider theme={theme}>
-            <UiConfigConsumer>
-              {(config) => (
-                <Provider store={configureStore(config)}>
-                  <I18nProvider i18n={i18n}>
-                    <BrowserRouter basename={BASE_PATH}>
-                      <Layout>
-                        <Route exact path="/" component={DiscoverPage} />
-                        <Route
-                          exact
-                          path="/dependency"
-                          component={DependenciesPage}
-                        />
-                        <Route
-                          exact
-                          path={['/traces/:traceId', '/traceViewer']}
-                          component={TracePage}
-                        />
-                      </Layout>
-                    </BrowserRouter>
-                  </I18nProvider>
-                </Provider>
-              )}
-            </UiConfigConsumer>
+            <SnackbarProvider>
```

Comment:
do we want to comment here why SnackbarProvider? it might not be obvious that this helps with things like archive

Reply:
Well we decided to move it here so that it could be reused in other parts of the UI and to decouple it from the trace page and archiving

Reply:
not arguing the rationale, just saying it is invisible unless looking at the comments. maybe a single-line comment about what this helps with?